src/Shared.Core/KoikatuAPIBase.cs

```diff
@@ -86,6 +87,7 @@ private void BaseAwake()
 
             EnableDebugLoggingSetting = Config.Bind("Debug", "Show debug messages", false, "Enables display of additional log messages when certain events are triggered within KKAPI. Useful for plugin devs to understand when controller messages are fired. Changes take effect after game restart.");
             RememberFilePickerFolder = Config.Bind("General", "Remember last folder in file pickers", true, "If true, file picker dialogs will remember the last opened folder and open already inside of it. If false, the dialogs will always open in the default folder.\n\nWarning: This setting only applies to plugins that use the file picker through this API.");
+            RememberFilePickerSaveLoad = Config.Bind("General", "Save remembered file picker folder between restarts", true, "Save and load last remembered file picker folders across game restarts");
```

Comment:
These two settings are too similar to each other and are going to get confusing. It would be better to have a single setting with an enum value with something like Disabled, Enabled, Reset on restart.